Yosemite/Yosemite/Model/Extensions/Order+LocalItems.swift

```diff
@@ -0,0 +1,43 @@
+/// Simple type to serve negative IDs.
```

Comment:
Move LocalIDStore and sanitizingLocalItems to Yosemite so it can be easily used in POS and normal orders